src/parse-templates.ts

```diff
@@ -66,14 +77,27 @@ function isEscaped(template: string, _offset: number | undefined) {
 export function parseTemplates(
   template: string,
   relativePath: string,
-  templateTag?: string
+  options?: string | ParseTemplatesOptions
```

Comment:
I think we mostly expect that this is private API (@dfreeman / @chriskrycho can hopefully confirm), I’d rather just change to taking options and not support the string fallback. 

What do you think?

Reply:
I much rather prefer to just change the API if possible, but of course only if we're not going to break anyone else. If someone can confirm, I can make that change quick.

Reply:
The only external consumer I _know_ of for this is the alpha-quality [`ember-template-imports` Glint environment](https://github.com/typed-ember/glint-environment-ember-template-imports/tree/initial-implementation) (which @ef4 is [working on upstreaming into Glint proper](https://github.com/typed-ember/glint/pull/301)).

I don't think it would be hard for us to absorb this there (or rather, just put out a release that requires a version of `ember-template-imports` that includes this change), but it raises the point that the public interface of this package is pretty nebulous, and there are clearly real-world use cases for using `parseTemplates` elsewhere.

Should we consider exporting this from the package root and making it more officially public?

Reply:
@rwjblue, @chriskrycho: based on Dan's comment above I went ahead and:
1) bumped the major version in `package.json`;
2) removed support for the string fallback.

Let me know if this works for you.

Reply:
@dfreeman Yeah, I think you’re right: having a public API for various tools will be important. I’m in support of making it public.

@ventuno that seems good to me. 👍🏼 Thanks!